apps/worker/src/app/workflow/usecases/add-job/add-job.usecase.ts

```diff
@@ -199,21 +216,16 @@ export class AddJob {
       return null;
     }
 
-    const job = await this.updateJob(response, command);
-
-    // Update the job digest directly to avoid an extra database call
-    command.job.digest = { ...command.job.digest, ...job } as IWorkflowStepMetadata;
-
     return response;
   }
 
-  private async updateJob(response: ExecuteOutput<DigestOutput>, command: AddJobCommand) {
```

Comment:
Changes below here are just renaming to make for a more accurate description of the method.